src/Workspaces/Core/Portable/CodeRefactorings/FixAllOccurences/FixAllState.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+using FixAllScope = Microsoft.CodeAnalysis.CodeFixes.FixAllScope;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings
+{
+    internal sealed class FixAllState : CommonFixAllState<CodeRefactoringProvider, FixAllProvider, FixAllState>
+    {
+        public override FixAllKind FixAllKind => FixAllKind.Refactoring;
+
+        public string CodeActionTitle { get; }
+
+        /// <summary>
+        /// Original selection span from which FixAll was invoked
+        /// </summary>
+        public TextSpan SelectionSpan { get; }
```

Comment:
what is this needed for?  Figuring out the containing member for fix-all-in-containing-member/type?  if so, doc plz.

Reply:
> what is this needed for? Figuring out the containing member for fix-all-in-containing-member/type? if so, doc plz.

Yes, also changed this to a private field now.